app/webpacker/components/Panel/pages/EditPersonRequestsPage/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Header } from 'semantic-ui-react';
+import TicketsList from '../../../TicketsList';
+import { ticketTypes, ticketStatuses } from '../../../../lib/wca-data.js.erb';
+
+export default function EditPersonRequestsPage() {
+  return (
+    <>
+      <Header>Open Tickets</Header>
+      <TicketsList
+        type={ticketTypes.edit_person}
+        status={ticketStatuses.edit_person.open}
+        sort="createdAt:desc"
+      />
+      <Header>Closed Tickets</Header>
+      <TicketsList
+        type={ticketTypes.edit_person}
+        status={ticketStatuses.edit_person.closed}
+        sort="createdAt:desc"
+      />
+    </>
```

Comment:
Can't you just loop over `ticketStatuses.edit_person` and create this copy-pasted "Header+TicketsList" thing on-the-fly?